tools/generate_sysroot.py

```diff
@@ -137,10 +137,10 @@ def generate_sysroot(sysroot: Path, api_version: float, arch: str, quiet=False):
   else:
     sys.exit('Unknown arch: ' + arch)
 
-  base_repo = 'http://download.tizen.org/snapshots/tizen/{}-base/latest/repos/standard/packages'.format(
-    api_version)
-  unified_repo = 'http://download.tizen.org/snapshots/tizen/{}-unified/latest/repos/standard/packages'.format(
-    api_version)
+  base_repo = 'http://download.tizen.org/snapshots/TIZEN/Tizen-{}/Tizen-{}-Base/latest/repos/standard/packages'.format(
```

Comment:
What's the difference between the directories `TIZEN` and `tizen`? Is `tizen` deprecated?

Reply:
tizen directory was deprecated.

Currently, the build system(quickbuild) is uploading new binaries in 'Tizen' directory.